test/remote/gateways/remote_worldpay_test.rb

```diff
@@ -494,6 +563,36 @@ def test_successful_authorize_with_3ds_with_gateway_specific_stored_credentials
     refute first_message.params['session_id'].blank?
   end
 
+  def test_successful_purchase_with_level_two_fields
+    assert response = @gateway.purchase(@amount, @credit_card, @options.merge(@level_two_data))
+    assert_success response
+    assert_equal true, response.params['ok']
+    assert_equal 'SUCCESS', response.message
```

Comment:
💬  Comment:

Is possible to check something in the response that relates to the specific level two data?

Reply:
no really